site/_posts/2018-01-28-troubleshooting-auth.md

```diff
@@ -0,0 +1,440 @@
+--
+title: "Troubleshooting ManageIQ Authentication"
+author: jvlcek
+date: 2018-01-28
```

Comment:
When would you like to publish this, next Monday perhaps? Please date accordingly, thanks!

Reply:
@cybette Thank you.

Can this be pushed today?

The render tool I use to develop will not allow for a date in the future. I will set it to a date closer to today.

Reply:
Date them yesterday (Jan 31) and I'll merge them today.